test/Microsoft.ML.Tests/TextLoaderTests.cs

```diff
@@ -877,5 +877,45 @@ public void TestTextLoaderNoFields()
                 Assert.StartsWith("Should define at least one public, readable field or property in TInput.", ex.Message);
             }
         }
+
+        public class BreastCancerInputModelWithKeyType
+        {
+            [LoadColumn(0)]
+            public bool IsMalignant { get; set; }
+
+            [LoadColumn(1), KeyType(10)]
+            public uint Thickness { get; set; }
+        }
+
+        public class BreastCancerInputModelWithoutKeyType
+        {
+            [LoadColumn(0)]
+            public bool IsMalignant { get; set; }
+
+            [LoadColumn(1)]
+            public uint Thickness { get; set; }
+        }
+
+        [Fact]
+        public void TestLoadTextWithKeyTypeAttribute()
+        {
+            var mlContext = new MLContext(seed: 1);
+            string breastCancerPath = GetDataPath(TestDatasets.breastCancer.trainFilename);
+
+            var data = mlContext.Data.CreateTextLoader<BreastCancerInputModelWithKeyType>(separatorChar: ',').Load(breastCancerPath);
+
+            Assert.True(data.Schema[1].Type.GetKeyCount() == 10);
```

Comment:
Instead of `Assert.True(something == expected)`, it is better to use `Assert.Equal(expected, actual)`. The reason is when it fails, xunit will print what the expected and actual values are, which helps figure out the issue, without having to re-run the test and debug.

Reply:
Great point @eerhardt, thanks for the review. Pushed the recommended change.